language/German/strings.po

```diff
@@ -925,35 +931,47 @@ msgstr "Die Bildschirmdrehung im Cocktailmodus wird nicht unterstützt.\n"
 
 #: src/frontend/mame/ui/info.cpp:135
 msgid "This system requires external artwork files.\n"
-msgstr ""
+msgstr "Dieses System erfordert externe Grafikdateien.\n"
 
 #: src/frontend/mame/ui/info.cpp:137
 msgid ""
 "This system was never completed. It may exhibit strange behavior or missing "
 "elements that are not bugs in the emulation.\n"
 msgstr ""
+"Dieses System wurde nie fertiggestellt. Es kann seltsames Verhalten zeigen oder "
+"es können Elemente fehlen, die keine Fehler in der Emulation sind.\n"
 
 #: src/frontend/mame/ui/info.cpp:139
 msgid ""
-"This system has no sound hardware, MAME will produce no sounds, this is "
-"expected behaviour.\n"
+"This system has no sound hardware, MAME will produce no sounds, this is expected "
+"behaviour.\n"
 msgstr ""
+"Dieses System hat keine Sound-Hardware, MAME wird keine Sounds produzieren, dies "
+"ist das erwartete Verhalten.\n"
 
 #: src/frontend/mame/ui/info.cpp:143
 msgid ""
 "\n"
 "THIS SYSTEM DOESN'T WORK. The emulation for this system is not yet complete. "
-"There is nothing you can do to fix this problem except wait for the "
-"developers to improve the emulation.\n"
+"There is nothing you can do to fix this problem except wait for the developers to "
+"improve the emulation.\n"
 msgstr ""
+"\n"
+"DIESES SYSTEM FUNKTIONIERT NICHT. Die Emulation für dieses System ist noch nicht "
+"vollständig. Sie können nichts tun, um dieses Problem zu beheben, außer darauf zu "
+"warten, dass die Entwickler die Emulation verbessern.\n"
 
 #: src/frontend/mame/ui/info.cpp:145
 msgid ""
 "\n"
-"Elements of this system cannot be emulated accurately as they require "
-"physical interaction or consist of mechanical devices. It is not possible to "
-"fully experience this system.\n"
+"Elements of this system cannot be emulated accurately as they require physical "
+"interaction or consist of mechanical devices. It is not possible to fully "
+"experience this system.\n"
 msgstr ""
+"\n"
+"Elemente dieses Systems können nicht genau emuliert werden, da sie physische "
+"Interaktion erfordern oder aus mechanischen Geräten bestehen. Es ist nicht "
+"möglich, dieses System vollständig zu erleben.\n"
```

Comment:
Please back out the rewrapped English messages.

Reply:
Fixed